.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: php
 
-php: [5.6, 7.1, hhvm]
```

Comment:
Oh, I didn't that you completely forbid phpspec releases before 4.0 now. Isn't it possible to support a constraint like `^3.0|^4.0`?